app/src/main/java/com/teamagam/gimelgimel/app/settings/dialogs/SetUsernameAlertDialogBuilder.java

```diff
@@ -70,6 +77,7 @@ public void onClick(DialogInterface dialog1, int which) {
 
     private void onPositiveButtonClicked() {
         setUsername(mInputEditText.getText().toString());
+        mOnFinishCallback.call();
```

Comment:
NPE can be thrown here.... 
Either check for null before call or initialize member with no-op action